src/processor/operator/update/set_executor.cpp

```diff
@@ -36,20 +59,40 @@ void MultiLabelNodeSetExecutor::set() {
             rhsPos = 0;
         }
         column->write(nodeID.offset, rhsVector, rhsPos);
+        if (propertyVector != nullptr) {
+            writeToPropertyVector(propertyVector, lhsPos, rhsVector, rhsPos);
+        }
     }
 }
 
 void RelSetExecutor::init(ResultSet* resultSet, ExecutionContext* context) {
     srcNodeIDVector = resultSet->getValueVector(srcNodeIDPos).get();
     dstNodeIDVector = resultSet->getValueVector(dstNodeIDPos).get();
     relIDVector = resultSet->getValueVector(relIDPos).get();
+    if (propertyPos.dataChunkPos != INVALID_DATA_CHUNK_POS) {
```

Comment:
Ditto here. We can init propertyVector to nullptr in RelSetExecutor's constructor.